test/config.js

```diff
@@ -0,0 +1,114 @@
+import path from 'path';
+import test from 'ava';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+
+const fixtureBasePath = path.resolve('test', 'fixtures', 'config');
+
+test('should return config only from home-directory when global binary used and `.np-config-json` in home-directory exists', async t => {
+	const homedirStub = sinon.stub();
```

Comment:
The stub is moved in the local method. So no need to run the test serially
(see (https://github.com/avajs/ava/issues/1066) and comment (https://github.com/sindresorhus/np/pull/456#discussion_r384717714)).